AB_environments/AB_sample.dask.yaml

```diff
@@ -0,0 +1,7 @@
+# Sample dask config file for A/B testing
+# It *must* be called AB_<name>.dask.yaml
+# and *must* be accompanied by AB_<name>.conda.yaml
+# Leave empty if you don't want to override anything.
+distributed:
+  scheduler:
+    worker-saturation: 1.2
```

Comment:
The alternative to forcing the developer to create an empty file every time was to ignore the file if it doesn't exist, but the risk of misspellings producing unexpected results was too high. Explicit is better than implicit.